src/tictactoe_server/app.clj

```diff
@@ -1,15 +1,18 @@
 (ns tictactoe-server.app
-  (:require [webserver.get]
-            [webserver.app :as app]
+  (:require [webserver.app :as app]
+            [webserver.get]
+            [tictactoe-server.util :as util]
             [clojure.java.io :as io]))
 
-(def most-recent-socket (atom nil))
 (defmulti route :uri)
-(defmethod route :default [request] (app/route @most-recent-socket request))
+(defmethod route :default [_])
+(defmethod route "/" [_] (slurp "assets/index.html"))
 
-(defn- copy [source socket] (io/copy source (.getOutputStream socket)))
-(defn handle [socket request]
-  (reset! most-recent-socket socket)
-  (doseq [response (route request)] (copy response socket)))
+(defn map-parameters [request]
+  (update request :parameters #(if % (util/parse-parameters %))))
 
-(defn initialize [] (app/initialize ["-d" "assets"]))
+(defn- write [response output-stream]
+  (doseq [r response] (io/copy r output-stream)))
```

Comment:
@trptcolin I am changing this, but Uncle Bob talks about using whitespace to group related closely functions. Is there a more concrete standard that? In this project I've been leaving out whitespace when the helper is used once and immediately.